swagger-def/ghc.yaml

```
@@ -4233,6 +4233,22 @@ definitions:
         items:
           $ref: 'definitions/Upload.yaml'
         type: array
+  ServiceRequestDocuments:
+    items:
+      $ref: '#/definitions/ServiceRequestDocument'
+    type: array
+  ServiceRequestDocument:
+    type: object
```

Properties should be added here with the mtoServiceItemId since it's a field on the model and will need to be returned from the database

```suggestion
     type: object
     properties:
      mtoServiceItemID:
        type: string
        format: uuid
```